apps/condo/package.json

```diff
@@ -113,6 +113,7 @@
     "less": "^3.12.2",
     "lodash": "^4.17.21",
     "minimatch": "^5.1.1",
+    "nanoid": "^3.3.4",
```

Comment:
We don’t need another one util … use existing

Reply:
We have no util to generate sequence unique id based of specific alphabet, like nanoid do. I can't use uuid() or open-id generators for those purposes